templates/index.html

```diff
@@ -1,4 +1,7 @@
 {% extends "layout.html" %}
 {% block body %}
-  <h1>Hello</h1>
+<h1>Scrobbler.io</h1>
+<p>Welcome to this simple Spotify to Last.fm scrobble service. What we do is to listen to your last played tracks on Spotify and scrobble them to your Last.fm account. Which makes your scrobbling easier, as you don't have to log into each of your units, or to miss out on scrobbles for when you are using Spotify Connect.</p>
```

Comment:
Cool. Could you warp the text to 80-120 characters? This makes reviewing easier.